### DISCUSSION TITLE
Can't change billing address

### DISCUSSION THREAD

**Discussion by MadL1me** [community/community#39740] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi!

Today I'm decided I want to support github library as sponsor. But because I'm was located in Russia, and it has US restrictions, because of my billing address, I cant be sponsor. I didn't find any way to change my billing address. It is possible for me to do so? Thank you

**Comment by mattbartley** [comment]
You should be able to edit your billing info here: [https://github.com/settings/billing/payment_information](https://github.com/settings/billing/payment_information)

**Reply by vusty** [reply]
It is impossible to change Billing information, there is no "Edit" button

**Reply by nazero3** [reply]
I am in turkey and there is no edit button

**Reply by Raflyromeo** [reply]
Same here. No Edit button on the billing page.
Might be related to region restrictions.

**Reply by nasirST** [reply]
Same with me I can't able to update my billing information. and no way to find to update.

**Comment by arseniy-nikitochkin** [comment]
I have the same problem. Billing address change is blocked, because my previous location was Russian Federation. Am I discriminated just for being born in Russia, no matter what country I live now? That's kinda humiliating, not to say insulting.😔

**Reply by SourceSurfer** [reply]
Согласен, тоже переехал и не работает, изменить нельзя

**Reply by karpen-dev** [reply]
В итоге исправили как-то?

**Comment by atlassian-me** [comment]
Can't change mine either (no edit button) and I'm in California.

**Comment by l11ama** [comment]
Good day! I can't change my billing information because I had billing address in Russian Federation. I don't live in Russia anymore and don't pay any taxes there. Also I submitted this form https://airtable.com/shrGBcceazKIoz6pY and I still haven't received a response. I don't know how to solve my problem and where should I leave this request. Hope you can help me. Otherwise it's a discrimination by your company.

Thank you!

**Reply by justerror** [reply]
Maybe you received a decision on the individual appellation after a month? I wonder if this procedure works at all.

**Reply by l11ama** [reply]
No, I haven't received any response after almost two months.

**Reply by codingjerk** [reply]
Any changes after 1 year?

**Reply by bmvalex** [reply]
I believe it's the same even now.

**Comment by Miskamyasa** [comment]


**Reply by l11ama** [reply]
No, still nothing. I've had to use new account for Copilot..

**Reply by dremdem** [reply]
> Was someone able to solve the problem? I am even already a citizen of another country and cannot use Copilot because of this problem.

Yes, I made a ticket, described a problem, and then the support solved the issue. So, I changed my billing info, assigned a new card and start using copilot whoaaa :)

**Comment by PadowYT2** [comment]
Hey, is this still not solved I guess? Same thing with me, lived in Russia, but no longer do, I moved out. Also locking **all** payments in Russia is really not necessary, you could just block Russian Cards, that's it

**Reply by PadowYT2** [reply]
Also to note - I've never been to Crimea, Donetsk & Luhansk. Only near Crimea (Sochi), Moscow & Siberia, and some other regions like Novosibirsk. And as stated under [GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls) those are the Russian regions only blocked, means every other region should be fine to buy stuff on Github

**Comment by sinbadxiii** [comment]
I join the problem :(

**Reply by jose8109** [reply]
help os falta

**Reply by jose8109** [reply]
is my personal money and my fone

**Comment by dremdem** [comment, marked as answer]
The same here. Can't change my billing info. Help please @github guys.

**Reply by dremdem** [reply]
I made an issue, described a problem, and then the support solved the issue. So, I changed my billing info, assigned a new card and start using copilot whoaaa :)

**Reply by l11ama** [reply]
Could you share where exactly you made an issue? And how long was the responding period?

**Reply by dremdem** [reply]
1. Go to [support](https://support.github.com/request/billing_and_payments?tags=hubberfy_billing_and_payments)
2. Billing problems
3. Describe the issue

>  how long 

1 day

**Reply by PadowYT2** [reply]
@MadL1me mark as solved this, I guess

**Reply by Miskamyasa** [reply]
I wrote to support. Attached image with my new GovID and voala. Payment is enabled again. Everything took 2 days

**Comment by arturheath** [comment]
i'll join the problem too in case it'll have a solution one day.
have to say to @github it's a shame you guys don't let people change their billing info

**Comment by Miskamyasa** [comment]


**Reply by codingjerk** [reply]
_Discrimination based on billing address_

**Reply by nazero3** [reply]
exactly

**Comment by cm4ker** [comment]
I have the same problem. I opened a ticket more than 2 weeks ago =(

**Reply by movalex** [reply]
Any progress on this?

**Reply by vectorsss** [reply]
I have the same issue, wrote the ticket yesterday. Waiting for the reply, hope they can solve the problem.

**Comment by glebignatieff** [comment]
+1 opened a ticket.

**Comment by Schum-io** [comment]
I have the same problem, ticket still open

**Comment by Tusenka** [comment]
Have the same issuue

**Reply by jose8109** [reply]
Ayuda hackeado en las billeteras virtuales help

**Comment by nikolay17538583** [comment]
had the same problem (previous ru address and not being able to change it) but made a ticket and GitHub support solved it in 3 days !

**Reply by predmaxim** [reply]
Можете подробнее описать что было и что сделали?

**Comment by jose8109** [comment]
Jose Luis Ovalle

**Comment by nazero3** [comment]
+1 I dont know why this is marked answered

**Comment by Salvehn** [comment]
Same here, changed payment info, but billing info is unchangeable

**Comment by spaytac** [comment]
**Update**: Support solved my problem within 24 hours. After a reset of my billing information, I was able to add them again - correctly this time 😄 
What I have witnessed: Auto-Fill functionality (chrome, in my case) will fill most of the Data correctly. Except for Country, in my case, Germany was auto-filled into the state/region Field.

I have the same problem. Never was in or near Belarus. Yesterday, I wanted to try Github Copilot. I had no billing profile, so I needed to create one. I used my browser's autofill functionality, checked the form briefly - never had problems until yesterday - and added my credit card as a payment method. I got a message (something similar to the following): "This functionality needs to be validated if you can use it.". This is because of adding payment information the first time. I thought.

Five minutes later, I received an email titled "Github and Trace Controls." I then realized that something had gone wrong. I checked my profile, and there it was, "Belarus". After realizing I would not solve this problem alone, I created a ticket. Now I am waiting.

**Comment by Doppelganger24** [comment]
I have the same problem, ticket still open

**Reply by ljw-struggle** [reply]
Is the problem solved?

**Comment by Roboyaroslava** [comment]
I wrote in support and still wait now...Hope Github support can reset billing information. Maybe in future they can developed special form for this situation, because I think it's frequently problem now..

**Comment by xsa-dev** [comment]
Can't change Billing Information. Please help! 🫣

**Comment by bezlant** [comment]
I have the same problem and github just left me hanging for weeks...

**Reply by LeTraceurSnork** [reply]
Did they reply to you yet?

**Reply by piratotoro** [reply]
Nothing. Over 2 months now. I had to create a new account.

**Comment by LeTraceurSnork** [comment]
Same here, as @bezlant said - I'm just waiting for noone to reply 🫤 Could you just discard the billing info in such cases then? @github

**Comment by AetherMagee** [comment]
Same here, opened a ticket to delete outdated billing info over a week ago. Still ghosted.

**Comment by thevadimspivak** [comment]
Same here, over two months theres no answer

**Comment by renbaoshuo** [comment]
Same here, opened a ticket but no response over two months. Submit the appeal form also didn't get any response. I'm living in China now.

**Comment by GameHackerTV** [comment]
Same here, opened a ticket but no response over 3+ months. 😭😭😭

**Comment by Wasdalt** [comment]
@github Can you tell me what to do now? I also opened a ticket to delete/change billing information. They haven't responded for over two days.

**Reply by movalex** [reply]
They wont reply. My ticket was submitted almost a year ago. The only way is to create a new account.

**Comment by ZenaY91** [comment]
@github now it's even impossible to open a ticket on https://support.github.com/request page . wtf?

**Reply by MrGeRoI** [reply]
It is at https://support.github.com/contact-next/product-selection/billing-and-payments/general-billing-and-payments . But they won't reply us anyway

**Comment by gosentetsu** [comment]
@github Same problem, please help to solve.

**Comment by MrGeRoI** [comment]
I hope they're not closing our tickets since they will restore back payment methods in Russia and Belarus after war ends

**Comment by Dorila24** [comment]
Same problem @github please help to solve it

**Comment by lskalinka** [comment]
I have the same problem @github please help

**Comment by kmishukov** [comment]
Same problem. @github

**Reply by kmishukov** [reply]
6 months, still nothing.

**Comment by MyNameIsHelloWorld** [comment]
I encountered the same problem. I hope @github can help me reset or restore it.

**Comment by mvalentsev** [comment]
Same problem. @github help, please.

**Reply by NatalliaKantser** [reply]
Same problem. https://github.com/github help, please.

**Comment by MyNameIsHelloWorld** [comment]
***@***.*** ***@***.***> 代表 karpen yapline ***@***.***>发送时间: 星期三, 六月 3, 2026 3:04:32 上午收件人: community ***@***.***>抄送: 1273064923 ***@***.***>; comment ***@***.***>主题: Re: [community/community] Can't change billing address (Discussion #39740)В итоге исправили как-то?—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today!

**Comment by karpen-dev** [comment]
Same problem. @github please help, i opened ticket a two month ago
